lib/uo_cli.rb

```diff
@@ -32,74 +32,194 @@
+
+            example_feature_file = "https://raw.githubusercontent.com/urbanopt/urbanopt-example-geojson-project/develop/example_project.json"
+            example_gem_file = "https://raw.githubusercontent.com/urbanopt/urbanopt-example-geojson-project/develop/Gemfile"
+            remote_mapper_files = ["https://raw.githubusercontent.com/urbanopt/urbanopt-example-geojson-project/develop/mappers/base_workflow.osw",
+                                   "https://raw.githubusercontent.com/urbanopt/urbanopt-example-geojson-project/develop/mappers/Baseline.rb",
+                                   "https://raw.githubusercontent.com/urbanopt/urbanopt-example-geojson-project/develop/mappers/HighEfficiency.rb"]
+            remote_weather_files = ["https://raw.githubusercontent.com/urbanopt/urbanopt-example-geojson-project/develop/weather/USA_NY_Buffalo-Greater.Buffalo.Intl.AP.725280_TMY3.epw",
+                                    "https://raw.githubusercontent.com/urbanopt/urbanopt-example-geojson-project/develop/weather/USA_NY_Buffalo-Greater.Buffalo.Intl.AP.725280_TMY3.ddy",
+                                    "https://raw.githubusercontent.com/urbanopt/urbanopt-example-geojson-project/develop/weather/USA_NY_Buffalo-Greater.Buffalo.Intl.AP.725280_TMY3.stat"]
+            
+            # Download files to user's local machine
+            remote_mapper_files.each do |mapper_file|
+                mapper_root, mapper_base = File.split(mapper_file)
+                mapper_download = open(mapper_file)
+                IO.copy_stream(mapper_download, File.join(mappers_dir_abs_path, mapper_base))
+            end
+            remote_weather_files.each do |weather_file|
+                weather_root, weather_base = File.split(weather_file)
+                weather_download = open(weather_file)
+                IO.copy_stream(weather_download, File.join(weather_dir_abs_path, weather_base))
+            end
+            gem_root, gem_base = File.split(example_gem_file)
+            example_gem_download = open(example_gem_file)
+            IO.copy_stream(example_gem_download, File.join(dir_name, gem_base))
+
+            feature_root, feature_base = File.split(example_feature_file)
+            example_feature_download = open(example_feature_file)
+            IO.copy_stream(example_feature_download, File.join(dir_name, feature_base))
+        end
     end
 
 
     # Perform CLI actions
-    if @options[:action_type] == "Clear"
-      puts "Deleting previous results from '#{scenario_file}'..."
-      scenario(scenario_file, actual_feature_file).clear
+    if @user_input[:project_folder]
+        create_project_folder(@user_input[:project_folder])
+        puts "\nAn example FeatureFile is included: 'example_project.json'. You may place your own FeatureFile alongside the example."
+        puts "Weather data is provided for the example FeatureFile. Additional weather data files may be downloaded from energyplus.net/weather for free"
```

Comment:
Maybe add:  'You will need to configure your mapper or your osw file to use the desired weather file'

Reply:
Good thought. I added a line about that.